server/liveblog/client_modules/client_modules.py

```diff
@@ -59,6 +63,53 @@ class ClientPostsService(PostsService):
     pass
 
 
+class ClientItemsResource(ItemsResource):
```

Comment:
Why do we need items here ?
Can we create multiple items comment ?
If no, can't we have just this `Comment` resource below ?